opm/parser/eclipse/EclipseState/Tables/TableManager.hpp

```diff
@@ -283,6 +285,21 @@ namespace Opm {
         }
 
 
+        template <class TableType>
```

Comment:
Does this really need to be templated?

Reply:
not really, i just blindly followed the code i monkey'd (the brine density).